### ISSUE TITLE
[Keyboard Navigation - Server Requests>Inspect]: Controls present in "Object" section under "Query" tab are not keyboard accessible.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66257]
### User Experience: 
Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls if interactive controls present on any page or pane are not keyboard accessible, hence fail to use their functionality. 

### Repro Steps:
Open #URL: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the "Server Requests Graphs". Expand the button and select the "Inspect" button. Select "Data" menu item. Navigate all the controls under the "Query" tab pane. Select the "Refresh" button. Select "+Expand all" button. Navigate to "Object " section. Verify the issue.

### Actual Result: 
Issue: Focus does not move to "Object" section and the controls present under it under 'Query' tab screen making it keyboard inaccessible.

### Expected Result:
  "Object" section and the controls present under it under 'Query' tab screen should receive keyboard focus and be accessible through keyboard.
Note:  The same issue exists with the same controls throughout the application. Tab stops is not working on the blade. The same issue exists with controls present under 'Toggle data source help' tab. Same issue exists under 'Filter by date' dropdown present under View rule screen.

### Severity
`SEV2`

### Area
`JSONFormatter (Saga Design System)`
